conditional/blueprints/intro_evals.py

```diff
@@ -31,11 +31,24 @@ def display_intro_evals(internal=False, user_dict=None):
     log = logger.new(request=request, auth_dict=user_dict)
     log.info('Display Intro Evals Listing')
 
-    # get user data
-    def get_fid_cm_count(member_id):
-        return len([a for a in FreshmanCommitteeAttendance.query.filter(
-            FreshmanCommitteeAttendance.fid == member_id)
-            if CommitteeMeeting.query.filter(CommitteeMeeting.id == a.meeting_id).first().approved])
+    def get_fid_cm(member):
```

Comment:
While you're here, could you write an actual doc comment (and since the return isn't a simple int anymore, maybe type annotations too?)